### PR TITLE
fix(plugin-chart-period-over-period-kpi): Blank chart when switching from BigNumberTotal

### PR DESCRIPTION
**Pull request by kgabryje**
### SUMMARY
When user creates and saves a Big Number chart and then switches to Big Number Period over Period, the chart appears blank. The reason for that is that Period over Period chart expects the values coming from `header_font_size` control to be pixels. However, the old Big Number chart uses proportions (0.2-0.6) in that control, so when the new chart inherits those values, the size in pixels is <1, so it seems blank.
This PR implements a mapping between proportion values used in Big Number and actual pixel size values expected by Period over Period chart.

Additionally, this PR moves the KPI chart to a common directory with other BigNumber plugins so that they can share to font size controls.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
1. Create a new Big number chart using the Vehicle Sales chart.
2. Save the chart
3. Switch to Big Number with Time Period Comparison" chart
4. Finally, update the chart.
5. Verify that the chart renders correctly

### TESTING INSTRUCTIONS

Before:

https://github.com/apache/superset/assets/15073128/2ad7c87a-1344-467d-bca4-67a5319b1958

After:

https://github.com/apache/superset/assets/15073128/f4b3e61c-bdd9-4b0c-9e61-fb15ce232c1a

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset-frontend/plugins/plugin-chart-echarts/src/BigNumber/BigNumberPeriodOverPeriod/transformProps.ts [hunk 1]
```diff
@@ -142,16 +143,15 @@ export default function transformProps(chartProps: ChartProps) {
...
     metric,
     metricName,
     bigNumber,
     prevNumber,
     valueDifference,
     percentDifferenceFormattedString: percentDifference,
     boldText,
-    headerFontSize,
-    subheaderFontSize,
+    headerFontSize: getHeaderFontSize(headerFontSize),
```
Comment (Antonio-RiveroMartnez) [inline superset-frontend/plugins/plugin-chart-echarts/src/BigNumber/BigNumberPeriodOverPeriod/transformProps.ts:153]: Should we make use of these utils in the other `BigNumber` viz too?

File: superset-frontend/plugins/plugin-chart-echarts/src/BigNumber/index.ts [hunk 2]
```diff
@@ -19,3 +19,4 @@
 
 export { default as BigNumberChartPlugin } from './BigNumberWithTrendline';
 export { default as BigNumberTotalChartPlugin } from './BigNumberTotal';
+export { default as PopKPIPlugin } from './BigNumberPeriodOverPeriod';
```
Comment (Antonio-RiveroMartnez) [inline superset-frontend/plugins/plugin-chart-echarts/src/BigNumber/index.ts:22]: Totally doable later on in a follow up PR, but could we align the component's name with it's directory name? i.e, from `PopKPIPlugin` to `BigNumberPeriodOverPeriod `

**Review by Antonio-RiveroMartnez — APPROVED** [review]
❤️ thanks for making these changes, I left a couple comments but nothing blocking.

File: superset-frontend/plugins/plugin-chart-echarts/src/BigNumber/BigNumberPeriodOverPeriod/transformProps.ts (hunk 1 above)
Comment (kgabryje) [reply superset-frontend/plugins/plugin-chart-echarts/src/BigNumber/BigNumberPeriodOverPeriod/transformProps.ts:153]: No - in those charts the actual font size is calculated dynamically based on chart's container width and height

File: superset-frontend/plugins/plugin-chart-echarts/src/BigNumber/index.ts (hunk 2 above)
Comment (kgabryje) [reply superset-frontend/plugins/plugin-chart-echarts/src/BigNumber/index.ts:22]: Oh right, I just copied this one from previous file

Comment (kgabryje) [reply superset-frontend/plugins/plugin-chart-echarts/src/BigNumber/index.ts]: Done

**Comment by sadpandajoe** [comment]
🏷️ preset:2024.8

**Pull request merged**